custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -15,7 +15,7 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-datatables/dist/plugins/bootstrap/datatables.bootstrap.min.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'leaflet/dist/leaflet.css' %}">
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'angular-busy/dist/angular-busy.min.css' %}"/>
-        <link type="text/css" rel="stylesheet" media="all" href="{% static 'perfect-scrollbar/css/perfect-scrollbar.min.css' %}"/>
+        <link type="text/css" rel="stylesheet" media="all" href="{% static 'utatti-perfect-scrollbar/css/perfect-scrollbar.css' %}"/>
```

Comment:
@orangejenny  is this fine to not have `min` I can't find the min version of it.

Reply:
Yeah, it's fine, it'll just add a tiny bit of time to dashboard load time.